package.json

```diff
@@ -91,6 +91,7 @@
     "node-fetch": "^2.6.0",
     "node-persist": "^3.0.5",
     "ora": "^4.0.3",
+    "package-lock.json": "^1.0.0",
```

Comment:
What is this for?

Reply:
i did pin the minimist transitive dependency but idk if that's related...

Reply:
yeah i removed it and it brought minimist back in

Reply:
https://www.npmjs.com/package/package-lock

looks like it lets you build from the package-lock and i had to use it to pin the dependency